external/payara-mp-tck/test.properties

```diff
@@ -1,6 +1,7 @@
 github_url="https://github.com/payara/MicroProfile-TCK-Runners.git"
 test_command="mvn --batch-mode clean verify"
 test_results="testResults"
+test_options="-pl !MicroProfile-OpenTracing/tck-runner,!MicroProfile-JWT-Auth,!MicroProfile-JWT-Auth/tck-arquillian-extension,!MicroProfile-JWT-Auth/tck-runner"
```

Comment:
At some point it may be interesting to try reincluding these, but definitely not needed as part of this PR